pkg/cmd/cli/cmd/logs.go

```diff
@@ -63,6 +69,7 @@ func NewCmdLogs(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Com
 		}
 		cmdutil.CheckErr(o.RunLog())
 	}
+	cmd.Flags().String("version", "", "View the logs of the deployment associated with the provided version for a deployment config")
```

Comment:
> Currently in deployments the object gets resolved in the server while in builds we do this in the client. What you are asking for can only be achieved by adding a version field in BuildLogOptions and doing the same thing that already happens in deployments (or we could continue to resolve the object in the client but that would still be incosistent). Is this fine?

Adding to `BuildLogOptions` and using to resolve client side would work.  Seems like we might want a `bc/logs` endpoint, but that's out of scope for this.